tests/bud/with-arg/Dockerfilefromarg

```diff
@@ -0,0 +1,11 @@
+ARG app_type
+ARG another_app_type
+
+FROM alpine as x
+RUN echo hello
+
+FROM ${app_type} as m
+RUN echo world
+
+FROM ${another_app_type} as final
+RUN echo hello
```

Comment:
Please also verify that an argument's value is expanded correctly if its definition supplies a default value, and that value isn't specified at the build command line.

Reply:
Added in the same containerfile above